IOPool/Input/src/PoolSource.cc

```diff
@@ -319,6 +365,8 @@ namespace edm {
         ->setComment("Size of ROOT TTree TBasket cache. Affects performance.");
     desc.addUntracked<unsigned int>("setRunNumber", 0U)
         ->setComment("If non-zero, change number of first run to this number. Apply same offset to all runs. Allowed only for simulation.");
+    desc.addUntracked<std::vector<unsigned int> >("setRunNumberForEachLumi", std::vector<unsigned int>())
```

Comment:
Talk with David Dagenhart about how to specify that this new parameter and the setRun can not be used simultaneously.